Jobs/MM_job.py

```diff
@@ -0,0 +1,324 @@
+#!/usr/bin/env python
+
+#   // INITIAL DESCRIPTION //
+"""Run Singlepoint Calculations"""
+
+#   // MEATDATA //
+__author__ = 'Alina Jansen'
+__date__ = '2024-09-19'
+
+#   // IMPORTS //
+
+#   Imports Of Existing Libraries
+import re
+import os
+import math
+import sqlite3
+import sys
+import subprocess
+import numpy as np
+
+#   Imports From Existing Libraries
+
+#   Imports Of Custom Libraries
+
+#   Imports From Custom Libraries
+from Logging.Logger import Logger
+from Generators._helper import filter_xyzq, _flatten
+from Generators.GeneratorGeometries import read_gmx_structure_header, read_gmx_structure_atoms, read_gmx_box_vectors, write_g96
+
+#   // TODOS & NOTES //
+#   TODO:
+#   - Add logger
+#   NOTE:
+
+#   // CLASS & METHOD DEFINITIONS //
+class MM():
+
+    '''
+    This Class Performs A Singlepoint Calculation
+    '''
+
+    def __init__(self, dict_input_userparameters, class_system, class_topology, class_pcf, str_directory_base) -> None:
```

Comment:
How about (less inflated variable names and type hints):

```python
import pathlib  # Maybe work with Path objects if you have paths
from collections.abc import MutableMapping

def __init__(self, parameters: MutableMapping, system: System, topology: Topology, pcf: PCF, directory: str | pathlib.Path) -> None:
```

Make sure to put in the right types for system, topology, etc.